partition/grub.go

```diff
@@ -63,28 +63,41 @@ func (g *grub) envFile() string {
 	return filepath.Join(g.Dir(), "grubenv")
 }
 
-func (g *grub) GetBootVar(name string) (string, error) {
+func (g *grub) GetBootVars(names []string) (map[string]string, error) {
+	out := map[string]string{}
```

Comment:
I'd suggest `out := make(map[string]string, len(names))` in this and the other non-mock `GetBootVars` (not that I'd mind if you did it in the mock ones as well)